src/app/log-in/log-in.component.html

```diff
@@ -5,9 +5,11 @@
   *ngIf="globalVars.inTab || globalVars.webview || globalVars.redirectURI"
 >
   <div class="margin-bottom--medium">
-    <h3 class="text--neutral-white">Log in to {{ globalVars.hostname }}</h3>
+    <img src="assets/image-head.svg" width="60" class="section--accounts__illustration"/>
+    <h4 class="text--neutral-white">Log in to {{ globalVars.hostname }}</h4>
   </div>
-  <div *ngIf="showAccessLevels">
+  <!-- <div *ngIf="showAccessLevels"> -->
```

Comment:
We need this conditional in here so we can only showAccessLevels if it's not a full access host.